src/bashi/results.py

```diff
@@ -827,3 +833,47 @@ def _remove_all_rocm_images_older_than_ubuntu2004_based(
             value_name2=HIPCC,
             value_version2=ANY_VERSION,
         )
+
+
+def _remove_unsupported_cuda_versions_for_ubuntu(
+    parameter_value_pairs_ranges: List[ParameterValuePair],
+    removed_parameter_value_pairs: List[ParameterValuePair],
+):
+    removed_parameter_value_pairs(
+        parameter_value_pairs_ranges,
+        removed_parameter_value_pairs,
```

Comment:
Instead calling the new function, you simply renamed the variable.

```suggestion
    parameter_value_pairs: List[ParameterValuePair],
    removed_parameter_value_pairs: List[ParameterValuePair],
):
    remove_parameter_value_pairs_ranges(
        parameter_value_pairs,
        removed_parameter_value_pairs,
```